tools/slo-generator/tests/unit/fixtures/migrate_format.py

```diff
@@ -1,169 +0,0 @@
-# Copyright 2019 Google Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#            http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# Script to migrate from old slo.json format (Stackdriver only) to new format
```

Comment:
Are we no longer supporting migrations?

Reply:
This was only used by one customer that used the SLO generator before we release the first version: should be safe to delete now since they migrated all their configs to the new format.